### DISCUSSION TITLE
Use arm64 cmake binaries on arm64 windows

### DISCUSSION THREAD

**Discussion by autoantwort** [microsoft/vcpkg#25444] [Ideas]
Currently the x86 version is downloaded on windows for arm while there exists an arm64 version of cmake. The x86 version works, but the arm64 version is faster.

**Comment by kafeg** [comment]
This is also most important, because now vcpkg uses x86 compiler. So compilation goes so long time and waste resources.

I building project in VS Code started from `ARM64 Native Tools Command Prompt for VS 2022Preview` with CMakePresets on native Windows 11 22H2 ARM64 with VS 2022 17.4 Preview and latest vcpkg for the 18.10.2022.

<img width="864" alt="image" src="https://user-images.githubusercontent.com/2333783/196409183-0bf992ab-4be7-4635-b92d-bc8b37042dba.png">

**Comment by kafeg** [comment]
I guess the problem is here: https://github.com/microsoft/vcpkg/blob/master/scripts/vcpkgTools.xml#L17
There is no platform specification.

Probably it can be solved manually, by the downloading cmake for arm64-windows and replacing folder which vcpkg deliver.
